unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -226,6 +250,11 @@ public enum Modifier
         FocusNotInInput = 0b0000010,
```

Comment:
Remember we had to answer how the modifiers work, each one of these deserves an explanation.